docs/CHANGELOG.asciidoc

```diff
@@ -10,11 +10,302 @@
 
 Review important information about the {kib} 8.0.0 releases.
 
+* <<release-notes-8.0.0-rc1>>
 * <<release-notes-8.0.0-beta1>>
 * <<release-notes-8.0.0-alpha2>>
 * <<release-notes-8.0.0-alpha1>>
 
 --
+[[release-notes-8.0.0-rc1]]
+== {kib} 8.0.0-rc1
+
+coming::[8.0.0-rc1]
+
+Review the {kib} 8.0.0-rc1 changes, then use the <<upgrade-assistant,Upgrade Assistant>> to complete the upgrade.
+
+[float]
+[[breaking-changes-8.0.0-rc1]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade, review the breaking changes, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+
+[discrete]
+[[breaking-118854]]
+.Splits package policy `upgrade` endpoint for Fleet
+[%collapsible]
+====
+*Details* +
+For package policy upgrades, the packagePolicy `upgrade` endpoint format supports a mutative upgrade operation (when `dryRun: false`) and a read-only dry run operation (when `dryRun: true`):
+
+[source,text]
+--
+ POST /package_policies/upgrade
+ {
+   packagePolicyIds: [...],
+   dryRun: false
+ }
+--
+
+For more information, refer to {kibana-pull}118854[#118854].
+
+*Impact* +
+The endpoint is now split into two separate endpoints:
+
+[source,text]
+--
+ POST /package_policies/upgrade
+ {
+   packagePolicyIds: [...]
+ }
+
+ POST /package_policies/upgrade/dry_run
+ {
+   packagePolicyIds: [...]
+ }
+--
+====
+      
+[discrete]
+[[breaking-119945]]
+.Removes APM jobs from Machine Learning
+[%collapsible]
+====
+*Details* +
+APM NodeJS and RUM Javascript anomaly detection job modules have been removed. For more information, refer to {kibana-pull}119945[#119945].
+
+*Impact* +
+When you upgrade to 8.0.0, you are unable to create and view the APM NodeJS and RUM Javascript jobs in Machine Learning.
+====
+
+[discrete]
+[[breaking-118300]]
+.Fails migrations for unknown types
+[%collapsible]
+====
+*Details* +
+Unknown saved objects types now cause {kib} migrations to fail. For more information, refer to {kibana-pull}118300[#118300].
```

Comment:
I'm not sure if this should be "saved objects types" or "saved object types". The latter sounds more correct to me, but I don't know what the rules are 🙂

Reply:
Also, instead of that PR, I think we should instead link to this issue, which contains much more context on the change: https://github.com/elastic/kibana/issues/107678 

The PR has very little info; most of the discussion that would be relevant to users is in the issue.